core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -75,23 +84,22 @@ public S3RestServiceHandler(@Context ServletContext context) {
   }
 
   /**
-   * @summary creates a bucket
    * @param bucket the bucket name
    * @return the response object
+   * @summary creates a bucket
    */
   @PUT
   @Path(BUCKET_PARAM)
-  @ReturnType("Response.Status")
+  @ReturnType("Void")
```

Comment:
Why do we want to revert the return type to `Void` ? I think it should be `Response.Status`

Reply:
The return type is the type of the body of the response, in this case (and others similarly), there is no body in the response, so I think it's `java.lang.Void` following the convention used in `PathsRestServiceHandler`.